homeassistant/helpers/collection.py

```diff
@@ -259,6 +261,30 @@ def attach_entity_component_collection(
     collection.async_add_listener(_collection_changed)
 
 
+@callback
+def attach_entity_registry_keeper(
```

Comment:
I prefer the name `attach_entity_registry_cleaner`

Reply:
https://github.com/home-assistant/home-assistant/pull/30844/commits/08d64a8f07733c0a22e065477c2248797b756a6d